Combined_NN_Model.py

```diff
@@ -1,70 +1,32 @@
 ### import the required modules and functions
-#TODO: Is this the correct Write_Performance()? --> NO, it's not. We should
-# update it with the parameters important for the NN in Utils.py
-# Same for NN_param_typecheck in NN_Utils.py
+
 import subprocess
 import time
 import sys
-
-from pycorenlp import StanfordCoreNLP
-
 from Utils import Write_Performance
 from config import *
-from reddit_parser import Parser
 from ModelEstimation import NNModel
 from transformers import BertTokenizer
 from NN_Utils import *
+from reddit_parser import Parser # Does the parser object need to be adjusted?
 
-# NOTE: Don't forget to set NN=True in defaults.py before running this file
-
-### Define the neural network object
-
-nnmodel=NNModel()
-
-
-### check key hyperparameters for correct data types
-
-nnmodel.NN_param_typecheck()
-
-### Write hyperparameters to file. Performance measures will be written to the
-# same file after analyses are performed
-
-Write_Performance()
-
-### call the parsing function
-
-# CoreNLP
-# create a connection to the CoreNLP server to retrieve sentiment
-# (requires CoreNLP_server.py in the same directory)
-subprocess.Popen(['java -mx6g -cp "*" edu.stanford.nlp.pipeline.StanfordCoreNLPServer --quiet'],
-               shell=True, cwd="./stanford-corenlp-4.0.0")
-time.sleep(5)  # wait for connection to the server to be established
-
+# QUESTION: Does the ID need to show up here in the functions too?
 theparser=Parser()
+
 # Create relevant folders
 theparser.safe_dir_create()
 
 # parse the documents
 theparser.Parse_Rel_RC_Comments()
```

Comment:
These are not needed in this file

Reply:
Got it, removing

Reply:
Not exactly. theparser=Parser() still should be there (the QUESTION on the line above it doesn't). I see it removed in the latest version. just safe_dir_create() and Parse_Rel_RC_Comments() need to be removed

Reply:
Oh, my mistake